p2p/host/resource-manager/limit.go

```diff
@@ -202,65 +270,65 @@ func (l *BaseLimit) GetMemoryLimit() int64 {
 }
 
 func (l *fixedLimiter) GetSystemLimits() Limit {
-	return &l.System
+	return &l.system
```

Comment:
Is there any reason to make these values private? When trying to integrate this on Kubo, we have some cases where we want to tweak final values after scaling, and we can't do that anymore if these values are private.

Reply:
Yes. We only want one way of changing things. I think fiddling with the `ConcreteLimitConfig` struct is tricky, since you have no defaults. You have to make sure everything is properly initialized.

If you want to tweak these values, the best way is to use a `PartialConfig` for the things you want to tweak and then Build it with a `ConcreteLimitConfig`. The readme has an example